dashboard/config/locales/data.en.yml

```diff
@@ -195,16 +195,16 @@ en:
         'bounce_11': "You can now attach blocks to the \"when run\" block. You have to attach the new \"launch new ball\" block in more than one place. Also, try changing the scene or the speed of the paddle and the ball and see how many points you can score before your opponent scores twice."
         'bounce_12': "Create your own Bounce game. You can change all the visuals and the rules. When you're done, click Finish to let friends try your game on their phones."
         'Custom_builder': "Drag blocks from the toolbox to the workspace to draw the level you wish to create. Press \"Run\" when you are ready."
-        'flappy_1': "Attach a block to the \"when click\" block, then press \"Run\". Click or tap the screen to move Flappy to the target."
-        'flappy_2': "Now we've added some ground.  Attach a block to the \"when hit the ground\" block.  Your code needs to 'END GAME' when Flappy crashes."
-        'flappy_3': "The \"when run\" block allows you to run code when your game starts.  Try setting the level speed and flapping to the target."
-        'flappy_4': "Flappy will fly through obstacles unless you write code to make him crash. Attach a block to the \"when hit an obstacle\" block. Your code needs to 'END GAME' when Flappy crashes into the first set of pipes."
-        'flappy_5': "Let's add a scoreboard, and count points when Flappy makes it past obstacles. Add to the \"when pass an obstacle\" block, and then score a point by flying Flappy through the first set of pipes."
-        'flappy_6': "See if you can still score a point after making Flappy flap either a smaller or larger amount."
-        'flappy_7': "Next, try changing the scene by attaching another block to the \"when run\" block."
-        'flappy_8': "You can set visuals anytime, not just when the game starts. Try setting a RANDOM scene when other events happen, for example, when passing an obstacle."
-        'flappy_9': "To mix things up, when hitting an obstacle, instead of ending the game try setting the score back to 0."
-        'flappy_11': "Create your own Flappy game. You can change all the visuals and all the rules, even the gravity. When you're done, click Finish to let friends try your game on their phones."
+        'flappy_1': "Attach the `flap` block to the `when click` event, then press \"Run\".<br/>Click or tap on the play space to move Flappy to the target."
```

Comment:
nit: I think we want to avoid mixing markdown and markup; replacing the `<br/>` with a couple newlines would be much prettier.